src/TuDa.CIMS.Web/Components/AssetList.razor

```diff
@@ -1,65 +1,201 @@
-<!-- grid with one column for items -->
+@using TuDa.CIMS.Shared.Entities
+<!-- table for asset list -->
 <div class="container-fluid text-center">
-    <div class="col">
-        @foreach (var item in Items)
+    <table class="table">
+        <thead class="align-middle">
+        <tr>
+            <th scope="col">Name</th>
+            <th scope="col">Typ</th>
+            <th scope="col">Raum</th>
+            <th scope="col">Notiz</th>
+            <th scope="col">ArtikelNr</th>
+            <th scope="col">Shop</th>
+            <th scope="col">CAS-Nr</th>
+            <th scope="col">Gefahren</th>
+            <th scope="col">Maßeinheit</th>
+            <th scope="col">Hersteller</th>
+            <th scope="col">SerienNr</th>
+            <th scope="col"></th>
```

Comment:
🌱 📌 We should think about which information we really want in this general overview. 
I make an issue for this.

Reply:
#47